src/main/java/io/stargate/sgv2/jsonapi/service/schema/tables/ApiVectorType.java

```diff
@@ -73,6 +73,23 @@ public static ApiVectorType from(int dimension, VectorizeDefinition vectorizeDef
     return new ApiVectorType(dimension, vectorizeDefinition);
   }
 
+  /**
+   * This method is only used for Dec hotfix. Currently, we don't support multiple vectorize service
+   * config. 1. If one doesn't have vectorizeDefinition (use user's own vector) and the other one
+   * have (use embedding service), it's ok 2. If both have vectorizeDefinition, only support if they
+   * are the same and have the same dimension
+   *
+   * @param v the other vector type to compare
+   * @return true if the two vector types are supported together
+   */
+  public boolean isTwoVectorColumnSupported(ApiVectorType v) {
+    if (this.vectorizeDefinition == null || v.getVectorizeDefinition() == null) {
```

Comment:
Maybe add a comment above; I think it means that one or both of Vector is not using Vectorization, and so there is no conflict?